assemblies/nexus-base-template/src/main/resources/overlay/etc/ehcache.xml

```diff
@@ -1,32 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ehcache xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://ehcache.org/ehcache.xsd"
-         updateCheck="false"
```

Comment:
The `updateCheck=false` is needed to prevent EHCache to "call home". It can be replaced by a system property too.

http://www.ehcache.org/generated/2.10.0/html/ehc-all/#page/Ehcache_Documentation_Set%2Fco-updt_configuring_the_update_checker.html%23wwconnect_header